lonabot/utils.py

```diff
@@ -43,25 +43,29 @@ class NoDeltaError(ValueError):
 _DUE_TIME = re.compile(fr'{_F}:{_F}(?::{_F})?')
 
 
-def parse_when(when, time_delta: TimeDelta, utc_now):
+def parse_when(when, time_delta: TimeDelta, utc_now, later_delta=None):
     # Returns `due` for either `delay` or `due`.
     due, text = parse_due(when, time_delta, utc_now)
     if due:
         return due, text
 
-    delay, text = parse_delay(when)
+    delay, text = parse_delay(when, later_delta)
     if delay:
         due = int(utc_now.timestamp()) + delay
         return due, text
 
     return None, None
 
 
-def parse_delay(when):
+def parse_delay(when, later_delta=None):
     iso = _parse_delay_iso(when)
     if iso:
         return iso
 
+    later = when.split(maxsplit=1)
+    if later[0] == 'later' and later_delta is not None:
```

Comment:
So this won't work if the user is angry and does `/REMIND LATER AGHHH`?

Reply:
Good question, probably not. Uppercase compare to "LATER"?

Reply:
This is now parsed case insensitive.